bin/jenkins.sh

```diff
@@ -68,14 +68,11 @@ function docker_version() {
 }
 
 function install_cli {
-    curl -O https://downloads.mesosphere.io/dcos-cli/install.sh
-    rm -rf dcos-cli/
-    mkdir dcos-cli
-    bash install.sh dcos-cli http://change.me --add-path no
-    source dcos-cli/bin/env-setup
-
-    # hack because the installer forces an old CLI version
-    pip install -U dcoscli
+    git clone git@github.com:dcos/dcos-cli.git
+
+    (cd dcos-cli && make env)
+    (cd dcos-cli/cli && make env)
+    source dcos-cli/cli/bin/env-setup-dev
```

Comment:
Users use to use `env-setup` when we had them install via python instead of via the native CLI.   It's a different set of deps.